### DISCUSSION TITLE
How to delete account if I was blocked?

### DISCUSSION THREAD

**Discussion by feschenko-nelya** [community/community#22004] [New to GitHub]
<p>Hello.</p>
<p>My account was blocked because I live in a specific region.<br>
I want to remove it. Delete action is available, I text my email and ‘delete my account’, but account is not removed. At the top of the web-site a warning in red appears: ‘It appears your account may be based in a U.S.-sanctioned region…’</p>
<p>How can I remove my account from GitHub?<br>
Is it possible?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there! I recommend reaching out to our <a href="https://support.github.com/request">Support team</a> to assist you here as they are the best contact in account-specific scenarios such as this one :slight_smile:</p>
